core/src/main/java/com/linecorp/armeria/internal/common/HttpObjectEncoder.java

```diff
@@ -40,6 +42,16 @@ default EventLoop eventLoop() {
         return channel().eventLoop();
     }
 
+    @Nullable
+    KeepAliveHandler keepAliveHandler();
+
+    default void keepAliveWrite(int id) {
```

Comment:
- I think this name is highly confusing. Please fine something better.
- This method doesn't have to be public if I understood correctly.

Reply:
Agreed, I removed this method and updated sub classes.